libs/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart-configuration.service.ts

```diff
@@ -124,7 +140,7 @@ export class FhirChartConfigurationService {
           x: this.timeline,
         },
         plugins: {
-          annotation: { annotations: [...annotations, this.todayDateVerticalLineAnnotation] },
+          annotation: { annotations: [...annotations, ...this.timeframeAnnotations] },
```

Comment:
Is this adding the timeframe annotations twice?

1. It adds them here directly from the instance property `this.timeframeAnnotations`
2. They are already in the `annotations` array because there were passed in as a parameter to `updateConfiguration()`